base/person.rb

```diff
@@ -21,6 +22,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    rental.person = self
+    @rentals << rental
+  end
```

Comment:
- [ ] the method for adding rentals should look a little different, please have another look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for examples

Reply:
Needs attention!